packages/microcosm/src/emitter.js

```diff
@@ -6,7 +6,7 @@
 
 import { isFunction } from './utils'
 
-export type Callback = (...args: *[]) => *
+export type Callback = (...args: any[]) => any
```

Comment:
Making a note to come back to this. We don't do any type checking of callbacks because they can accept anything and we don't handle their return value.